Cython/Utility/ObjectHandling.c

```diff
@@ -2727,7 +2727,7 @@ static CYTHON_INLINE PyObject* __Pyx_PyObject_CallNoArg(PyObject *func); /*proto
 
 static CYTHON_INLINE PyObject* __Pyx_PyObject_CallNoArg(PyObject *func) {
     PyObject *arg = NULL;
-    return __Pyx_PyObject_FastCall(func, (&arg)+1, 0 | __Pyx_PY_VECTORCALL_ARGUMENTS_OFFSET);
+    return __Pyx_PyObject_FastCall(func, (&arg), 0 | __Pyx_PY_VECTORCALL_ARGUMENTS_OFFSET);
```

Comment:
`__Pyx_PY_VECTORCALL_ARGUMENTS_OFFSET` is telling Python "there is space at `(&arg)-1` so feel free to write into it to speed up method calls". Therefore what you've done will break some calls.

I think you need to allocate an array of two `PyObject*` instead.

Reply:
Force pushed the fix. Thank you for pointing this out.